<launch>
  <node name="squirrel_fixate_on_3D_point_node"
        pkg="squirrel_attention"
        type="fixate_on_3D_point_node"
        required="true" output="screen">

        <remap from="/pan_topic" to="/pan_controller/relative_command"/>
        <remap from="/tilt_topic" to="/tilt_controller/relative_command"/>
        <remap from="/pan_service" to="/pan_controller/set_relative_position"/>
        <remap from="/tilt_service" to="/tilt_controller/set_relative_position"/>

  </node>

</launch>
